### PR TITLE
React Capstone: Check Current Weather Mobile Webapp

### PR DESCRIPTION
**Pull request by binyamolango**
# Hi Reviewer 👋

> Here are the changes I implemented to the project

- Selected the API I am going to use.
- Built file structure for my React app.
- Prepared routes and navigation in my app.

- Made sure that a user can display a list of items and filter them. Data should come from the API.
- Created the tests for the application.

- Styled my components to match the design provided.
- Deployed the project and tested for final details.
- Recorded a video for my project.
- Created a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
     <h1>📗 Table of Contents</h1>
   </summary>
 
-- [Space Travelers' Hub](#space-travelers-hub)
-- [📖 Space Travelers' Hub ](#-space-travelers-hub-)
+- [Check Weather](#Check-Weather)
+- [📖 Check Weather ](#-Check-weather-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
```
Comment (TadesseDev) [inline README.md:18]: - [ ] Dear @binyamolango, Please keep in mind, The README file is expected to contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please add the acknowledgment.

File: README.md [hunk 2]
```diff
@@ -56,29 +56,28 @@
...
+- **Fetch and render the weather conditions**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Soon]()
+- [Watch live](https://check-currect-weather.onrender.com/)
```
Comment (TadesseDev) [inline README.md]: - [ ] For the live demo section, We need to make sure the app is deployed on Render, Netlify, or GitHub Pages. Also, we need to make sure all the functionalities are working well on the live version. Currently, I cannot open your live version and see the deployed app. Please make sure everyone can open and use your app live on the internet.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @binyamolango 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still naeed to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] You did great with the test's ✅ 
 - [x] Good job overall following the template design ✅
 - [x] You have written a clean code ✅
 - [x]  Most of the requirement has been meet. ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binyamolango** [comment]
@TadesseDev Thank you for the insightful suggestions. I implemented the required changes.

File: README.md (hunk 2 above)
Comment (Whoistolu) [reply README.md]: - Well resolved 👍🏽

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md:18]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @binyamolango 

Great job implementig the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ] [OPTIONAL] Kindly note that for solo capstone projects, the correct flow for requesting reviewers is Gitflow which means you open a PR wanting to merge from your final feature branch to `dev` branch while for group capstones, the correct Github flow is to open a PR wanting to merge into `main` from `dev` branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binyamolango** [comment]
@Whoistolu Thank you for reviewing, and approving my project and also for the insightful message on implementing the correct Gitflow. Thank you again!

**Pull request merged**